apps/minaintyg/src/components/error/TechnicalIssueAlert.tsx

```diff
@@ -0,0 +1,11 @@
+import { IDSAlert } from '@frontend/ids-react-ts'
+import { ReactNode } from 'react'
+
+export function TechnicalIssueAlert({ children }: { children: ReactNode }) {
```

Comment:
Test for component is missing

Reply:
It's tested in both `CertificateListPage` and `CertificatePage` where it's used. :)